test/electionguard/test_encrypt.cpp

```diff
@@ -76,7 +76,7 @@ TEST_CASE("Encrypt PlaintextBallot with EncryptionMediator against constructed "
     auto election = ElectionGenerator::getFakeElection();
     auto metadata = make_unique<InternalElectionDescription>(*election);
     auto context = ElectionGenerator::getFakeContext(*metadata, *keypair->getPublicKey());
-    auto device = make_unique<EncryptionDevice>(12345UL, "Location");
+    auto device = make_unique<EncryptionDevice>(12345UL, 23456UL, 34567UL, "Location");
```

Comment:
similar to python, this should move into generators/mocks

Reply:
why is this comment here but it's not moved?

Reply:
changed my mind